.travis.yml

```diff
@@ -74,8 +74,8 @@ script:
     if [ $BUILD_EXTERNAL == "0" ]; then
       mkdir llvm-spirv
       mv * llvm-spirv
-      git clone https://git.llvm.org/git/llvm.git/ --depth 1
-      mv llvm-spirv llvm/tools/llvm-spirv
+      git clone https://github.com/llvm/llvm-project --depth 1
+      mv llvm-spirv llvm-project/llvm/tools/llvm-spirv
```

Comment:
I think we better have it at `llvm-project/llvm-spirv`, like other llvm projects.

Reply:
I doubt that's going to work for `BUILD_EXTERNAL=0` builds, because cmake will not pick up the llvm-spirv project.  This is normally done by `add_llvm_implicit_projects()` in `llvm-project/llvm/tools/CMakeLists.txt`.

But let me push your suggestion as a separate commit to see if it works or not.

Reply:
Failed as expected:
```make: *** No rule to make target 'llvm-spirv'.  Stop.```
So I'll change it back to `llvm-project/llvm/tools/llvm-spirv`

Reply:
According to this page https://llvm.org/docs/CMake.html I think we should specify the following cmake options:
```
LLVM_EXTERNAL_LLVM_SPIRV_SOURCE_DIR=../llvm-spirv
LLVM_EXTERNAL_PROJECTS="llvm-spirv"
```

Reply:
AFAIK, in addition to that you need also "enable" this external project.

-DLLVM_ENABLE_PROJECTS="llvm-spirv"

You can take a look how we build `llvm-spirv` with monorepo in SYCL project here:
https://github.com/intel/llvm/blob/sycl/sycl/doc/GetStartedWithSYCLCompiler.md#build-the-sycl-compiler-and-runtime

Reply:
Got it working now, thanks for your suggestions.